packages/core/parcel-bundler/src/transforms/postcss.js

```diff
@@ -54,11 +54,8 @@ async function getConfig(asset) {
   }
 
   if (asset.options.minify) {
-    let [cssnano, {version}] = await Promise.all(
-      ['cssnano', 'cssnano/package.json'].map(name =>
-        localRequire(name, asset.name).catch(() => require(name))
```

Comment:
`.catch(() => require(name))`
What was that part for?

Reply:
not sure, seems unnecessary. do you remember @DeMoorJasper?